### PR TITLE
Migrate common components to new i18n framework

### PR DESCRIPTION
**Pull request by bryk**
We'll not actively migrate everything, but this is a PR to set grounds
and precedence for others to refer to.

### CODE REVIEW

**Comment by bryk** [comment]
@taimir Can you review this?

**Comment by taimir** [comment]
I'll skip over it on github, but I don't have a dev environment around me today to compile and test. That I can do tomorrow earliest.

**Comment by bryk** [comment]
Sure, no problems.

File: src/app/frontend/common/components/zerostate/zerostate.html [hunk 1]
```diff
@@ -17,9 +17,15 @@
...
+      <div class="kd-zerostate-title">[[There is nothing to display here|Title text which
+        appears on zero state view.]]</div>
       <div class="kd-zerostate-text">
-        <span ng-bind-html="::$ctrl.i18n.MSG_ZERO_STATE_TEXT"></span>
+        <span>
+          [[You can <a href="{{::$ctrl.getStateHref()}}">deploy a containerized app</a>,
+          select other namespace or
+          <a href="http://kubernetes.io/docs/user-guide/ui/" target="_blank">take the Dashboard Tour
+          <i class="material-icons kd-zerostate-icon">open_in_new</i></a>
+          to learn more| Description for a zero state page.]]</span>
```
Comment (taimir) [inline src/app/frontend/common/components/zerostate/zerostate.html]: The variables feel very natural now! (to reiterate my excitement from before :D) That's probably the best part for me personaly, great stuff.

File: i18n/messages-ja.xtb [hunk 2]
```diff
@@ -1009,4 +1009,29 @@
...
+	<translation id="8022917849813090234" key="MSG_PODLIST_PODCARDLIST_9" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Tooltip for pending pod card icon">This pod is in a pending state.</translation>
+	<translation id="7339906504794526992" key="MSG_PODLIST_PODCARDLIST_10" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Label for logs tooltip">Logs</translation>
+	<translation id="403630050812245993" key="MSG_PODLIST_PODCARDLIST_11" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Name of the pod resource">Pod</translation>
+	<translation id="6710159274365919294" key="MSG_PODLIST_PODCARDLIST_12" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Name of the pod resource">Pod</translation>
```
Comment (taimir) [inline i18n/messages-ja.xtb]: OK, just this problem: now that the variables are named differently, the existing japanese translations are not picked up. So someone will need to move them manually in the `messages-ja.xtb` file to the new variables and delete the old ones (which are still there with the old names). E.g. `MSG_ACTION_BAR_CREATE_ACTION_TOOLTIP` is now `MSG_COMMON_COMPONENTS_ACTIONBAR_ACTIONBAR_0`.

**Comment by maciaszczykm** [comment]
@taimir Remember to use review system when reviewing :) Status should be `changes requested` right now, and after LGTM `approved` should be there.

File: i18n/messages-ja.xtb (hunk 2 above)
Comment (bryk) [inline i18n/messages-ja.xtb]: Oh, yeah. Is this something we should do in this PR or should we call out our Japanese friends?

**Comment by taimir** [comment]
Wait, but I did choose `add your review` and then `request changes` when I posted my last comment. And now it says `taimir suggested changes 19 minutes ago`. Is that now how it should be?

Also when I LGTMed before, I chose `add your review` and then `approve`, and on my comment it says `taimir approved these changes 21 hours ago`. 

What am I doing wrong?

**Comment by maciaszczykm** [comment]
Hmm, then it looks strange on my site:

image

It says, that there are none reviews yet.

**Comment by floreks** [comment]
Probably this is the case. From GitHub doc:

> Repository administrators can require that all pull requests receive at least one approved review from someone with write or admin permissions before they're merged into the default branch. For more information, see "About protected branches."

File: i18n/messages-ja.xtb (hunk 2 above)
Comment (taimir) [inline i18n/messages-ja.xtb]: Well I think it's ok to do it in a separate patch. WDYT?

**Comment by taimir** [comment]
Yep, we had this already in another PR that I reviewed. I can just review, but a final approval from an owner / admin is needed as well.

**Comment by maciaszczykm** [comment]
@floreks Okay.

@taimir Never mind my previous comment.
